src/components/color-picker/gradient-preview/GradiantPreview.tsx

```diff
@@ -0,0 +1,26 @@
+import styled from '@emotion/styled';
```

Comment:
Typo in the file name.

Reply:
Also please move the file to the `gradient-select` folder as it's an internal component.